### PR TITLE
feat(grants): login screen - update registration section text and links

### PR DESCRIPTION
**Pull request by lsr-explore**
## Description

Update the bottom registration section to match Figma 
image

Approval from UX
[Slack request post](https://usdigitalresponse.slack.com/archives/C04UU76HR9D/p1728428403367969)
[Slack approval](https://usdigitalresponse.slack.com/archives/C04UU76HR9D/p1728435087330769)

Test Coverage 

File                   | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s

**Before**
LoginView.vue |   34.61 |    33.33 |   22.22 |      36 | 1,22,57,109-136

**After**
LoginView.vue |   59.25 |    38.88 |   44.44 |   61.53 | 23,136-158

## Screenshots / Demo Video

App
image

Ticket

New Jersey

href="https://forms.office.com/pages/responsepage.aspx?id=0cN2UAI4n0uzauCkG9ZCp7ITdp07WUhBmfHGHVfxwctUNVFJNFNPUllWSTBQNFZCQlMyMjlWWU5NUiQlQCN0PWcu&amp;route=shorturl" 

Nevada
href="https://forms.office.com/pages/responsepage.aspx?id=5kCj5J64aE6OqhVE0nA5gBT1YIeXbrtMjy6yqM5ILm1UQVQwT0xOT1QxU0dNU1BDN05WWDdaVFRBRS4u&route=shorturl"

### Automated and Unit Tests
- [ X] Added Unit tests

### Manual tests for Reviewer
- [ ] Added steps to test feature/functionality manually

## Checklist
- [ ] Provided ticket and description
- [ ] Provided screenshots/demo
- [ ] Provided testing information
- [ ] Provided adequate test coverage for all new code
- [ ] Added PR reviewers

### CODE REVIEW

File: packages/client/src/views/LoginView.vue [hunk 1]
```diff
@@ -1,78 +1,101 @@
...
-        <div>
-          <label for="email">Email</label>
-          <input
-            id="email"
-            v-model="email"
-            class="form-control mb-4"
-            name="email"
-            placeholder="Email address"
-            autofocus
+  <main>
```
Comment (lsr-explore) [inline packages/client/src/views/LoginView.vue:2]: Note to review:  Fixing an accessibility issue - all content should be within a main landmark.

**Review by phm200 — APPROVED** [review]
thank you for adding the test coverage and for looking out for a11y issues

if we upgrade vitest to a more recent version (in a future PR), we could use "getByRole" so that our tests are a bit more robust and use a11y role and name: https://main.vitest.dev/guide/browser/locators#getbyrole

**Comment by lsr-explore** [comment]
> thank you for adding the test coverage and for looking out for a11y issues
> 
> if we upgrade vitest to a more recent version (in a future PR), we could use "getByRole" so that our tests are a bit more robust and use a11y role and name: https://main.vitest.dev/guide/browser/locators#getbyrole

Awesome @phm200 !   I was looking for something like that - I had used testing library for react on other projects.   I'll play around with upgrading later next week.

I plan to propose that we include https://www.npmjs.com/package/eslint-plugin-vuejs-accessibility as an eslint plugin.

**Comment by phm200** [comment]
> > thank you for adding the test coverage and for looking out for a11y issues
> > if we upgrade vitest to a more recent version (in a future PR), we could use "getByRole" so that our tests are a bit more robust and use a11y role and name: https://main.vitest.dev/guide/browser/locators#getbyrole
> 
> Awesome @phm200 ! I was looking for something like that - I had used testing library for react on other projects. I'll play around with upgrading later next week.
> 
> I plan to propose that we include https://www.npmjs.com/package/eslint-plugin-vuejs-accessibility as an eslint plugin.

I also have a lot of experience with testing library and React. I haven't used the a11y plugin in eslint, but would be great to keep on top of a11y issues. Would be good to see how strict it is before turning on. From past work, when we enable new rules, we first fix all the rules in one or many PRs before turning on the rule in settings; just to reduce churn for others

**Pull request merged**